src/SerialLoops/Editors/Editor.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Eto.Forms;
+using Eto.Drawing;
+using SerialLoops.Lib.Items;
+
+namespace SerialLoops.Editors
+{
+    internal abstract class Editor : TabPage
+    {
+
+        public readonly ItemDescription Description;
```

Comment:
Make sure to include accessors here:

```suggestion
        public readonly ItemDescription Description { get; private set; }
```